docs/HOL_Continuous-Integration/README.md

```diff
@@ -1,215 +1,38 @@
-your Visual Studio Team Services account. While pushing, use the user name (secondary) and password you have created when enabling alternate authentication credentials earlier in the lab.
-
-	cd PartsUnlimitedMRP/
-
-	git remote add vsts <url_to_repository>
-
-	git push -u vsts --all
-	
-![](<media/push_to_vsts.png>)
-
-**NOTE:** we added the Visual Studio Team Services repository as a remote named **vsts**, so we need to
-push to that remote in the future for our changes to appear in our Visual Studio Team Services
-repository.
-
-**4.** Your Visual Studio Team Services account should now have a copy of the PartsUnlimitedMRP
-application:
-
-![](<media/mrp_in_vsts.png>)
-
+**Create a Continuous Integration Build:** In this step, you will create a build definition in Visual Studio Team Services that will be triggered every time a commit is pushed to your repository in Visual Studio Team Services. 
  
+### 1. Create a Continuous Integration Build
 
-### 1. Set up Linux virtual machine in Azure as the Build Agent
-
-The application is written in Java, so we are going to use a Linux machine to
-build it.
-
-HINT: You can use OSX, Ubuntu, or a RedHat VM to run the agent. This guide is based on an Ubuntu VM.
-
-**1.** Go to <https://portal.azure.com>
-
-**2.** Click the **New** button in the top left corner of the Azure portal
-
-![](<media/azure_new_resource.png>)
-
-**3.** Select **Compute**, then **Ubuntu Server 14.04 LTS**, and click
-**Create**
-
-![](<media/new_ubuntu_vm.png>)
-
-**4.** Enter a **host name**, **user name**, and **password**. Click **Create**
-
-![](<media/create_vm_opts.png>)
-
-**5.** Once the machine has been created select the **tile for the machine**,
-click on **Endpoints**, and note the public **DNS name** and the **public port**
-that was chosen for SSH access.
-
-![](<media/ssh_details.png>)
-
-**6.** Connect via SSH to the new Linux machine using the **public DNS name** and **public port** from step five.
-
-	ssh <user>@<public_dns> -p <public_port>
-
-![](<media/create_ssh.png>)
-
-**7.** If you are on **Ubuntu 14.04**, run these commands; otherwise, **ignore
-this step**:
-
-Press [ENTER] to continue when asked after the first command.
-
-	sudo add-apt-repository ppa:openjdk-r/ppa
-
-	sudo apt-get update
-
-**8.** Copy and paste the following snippet to **run these commands**:
+>NOTE: Ensure that you have an existing PartsUnlimitedMRP team project that also contains the Git repository cloned from GitHub. If not, complete the "Set up Parts Unlimited MRP" lab before going through this lab. 
 
-	# Install git client
-	sudo apt-get install git -y
+>In this lab, we will be using the Hosted agent located in Visual Studio Team Services. If you would like to use an on-premises cross-platform agent (Azure subscription needed), you can follow instructions for setting an agent up [with this link](https://github.com/Microsoft/vsts-agent/blob/master/README.md). 
```

Comment:
The link redirect to the repo for VSTS Agent not to the instructions for setting an agent up....